aws.go

```diff
@@ -6,17 +6,58 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/percona/rds_exporter/config"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
-	PeriodSeconds = 60
-	DelaySeconds  = 600
-	RangeSeconds  = 600
+	Period = 60 * time.Second
+	Delay  = 600 * time.Second
+	Range  = 600 * time.Second
+
+	latencyDesc = prometheus.NewDesc(
+		"rds_latency",
+		"The difference between the current time and timestamp in the metric itself",
+		[]string{"instance", "region"},
+		map[string]string(nil),
+	)
 )
 
+type Scrape struct {
+	// params
+	instance  config.Instance
+	collector *Collector
+	ch        chan<- prometheus.Metric
+
+	// internal
+	svc     *cloudwatch.CloudWatch
+	labels  []string
+	latency *Latency
+}
+
+func NewScrape(instance config.Instance, collector *Collector, ch chan<- prometheus.Metric) *Scrape {
+	// Create CloudWatch client
+	sess := collector.Sessions.Get(instance)
+	svc := cloudwatch.New(sess)
+
+	// Create labels for all metrics
+	labels := []string{}
+	labels = append(labels, instance.Instance, instance.Region)
```

Comment:
```go
labels := []string{instance.Instance, instance.Region}
```

Reply:
Changed in https://github.com/percona/rds_exporter/pull/4